docs/docs/theory_2.md

```diff
@@ -44,10 +44,94 @@ $$
 \rho_{ij}(t) = e^{-\frac{i}{\hbar}\left(\lambda_i-\lambda_j\right) t}\rho_{ij}(0)
 $$
 
-This way we can see that the equations are completely decoupled. Coefficients on the diagonal of the density matrix don't change, while off-diagonal coefficients gain a phase factor at a constant rate that is dependent on the differences between the Hamiltonian eigenvalues. This method gives us the exact evolution of the system and perfectly preserves unitarity. The downside of it is that it requires a full diagonalization of the Hamiltonian. However, many spin systems that we are interested in are relatively small, and one single diagonalisation for each of them isn't a big deal. Cheaper, more approximate methods might be implemented in the future, but at the moment, MuSpinSim can easily carry out calculations on systems that don't exceed nine or ten spins on a laptop in a few minutes.
+This way we can see that the equations are completely decoupled. Coefficients on the diagonal of the density matrix don't change, while off-diagonal coefficients gain a phase factor at a constant rate that is dependent on the differences between the Hamiltonian eigenvalues. This method gives us the exact evolution of the system and perfectly preserves unitarity. The downside of it is that it requires a full diagonalization of the Hamiltonian. However, many spin systems that we are interested in are relatively small, and one single diagonalisation for each of them isn't a big deal. For a cheaper, more approximate method see [Celio's Method](#celios-method).
 
 > **For developers:** time evolution of a system is handled by the `.evolve()` method of the `Hamiltonian` class.
 
+#### A faster method
+
+For cases when $\frac{B}{T} \rightarrow 0$, for instance when we are simulating a zero field experiment or one where we take $T\rightarrow = \infty$, we no longer need the density matrix and automatically employ a faster method. To do we can first note that we will be dealing with a spin-polarised muon with various other unpolarised spins. The density matrix at $t=0$ for the muon polarised along ${\hat n}$ can be written as
+
+$$
+ \rho_\mu (t=0) = \frac{1}{2}(1 + \sigma_\mu^{\hat n}),
+$$
+
+and hence the density matrix of the full system is, (defining $d =\prod_{i \neq 0} 2I_i + 1$ as the dimension of the Hilbert space without the muon), 
```

Comment:
```suggestion
and hence the density matrix of the full system is (defining $d =\prod_{i \neq 0} 2I_i + 1$ as the dimension of the Hilbert space without the muon)
```
Not important, but having the comma before and after the brackets tripped me up when reading (either would be fine but I don't think we need both).